.github/workflows/python-wheel-build.yaml

```diff
@@ -0,0 +1,62 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Build Python Wheel
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        python-version: ['3.8', '3.9', '3.10']
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }} on ${{ matrix.os }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Update pip and setuptools
+      run: |
+        pip install --upgrade pip
+        pip install --upgrade setuptools wheel
+    - name: Run build_wheel script
+      run: python scripts/build_wheel.py
+    - name: Save wheel artifact
+      uses: actions/upload-artifact@v3
+      with:
+        name: wheel-${{ matrix.os }}-${{ matrix.python-version }}
+        path: dist/adatest-0.3.4-py3-none-any.whl
```

Comment:
Does this mean we need to update this file every time the version of adatest changes? If so we should probably load this from adatest.__version__ instead

Reply:
Thanks for pointing this out. You're right that we wouldn't want to update this workflow every time; that would defeat the point of it.